config-templates/redhat/odf-local/4.7/deployment.yaml

```diff
@@ -324,8 +324,8 @@ items:
               allowPrivilegeEscalation: false
             resources:
               limits:
-                cpu: 100m
-                memory: 200Mi
+                cpu: 500m
+                memory: 500Mi
```

Comment:
Why are we not updating image for 4.7?

Reply:
4.7 is deprecated and secure PR has raised and will push the changes to prod in few days